### PR TITLE
Add the guessed default to the read-module prompt

### PR DESCRIPTION
**Pull request by quodlibetor**

### CODE REVIEW

File: ansible-doc.el [hunk 1]
```diff
@@ -159,6 +159,9 @@ buffer-local wherever it is set."
          (default (if (or (null modules) (member symbol modules))
                       symbol
                     nil))
+         (prompt (if default
+                     (concat prompt " [" default "]: ")
```
Comment (lunaryorn) [inline ansible-doc.el]: Please use ` (default …): ` for the prompt, for consistency with other similar prompts in Emacs (e.g. `describe-function`).

Comment (quodlibetor) [inline ansible-doc.el]: Updated, thanks.

**Comment by lunaryorn** [comment]
Merged, thank you!

**Comment by quodlibetor** [comment]
Thank you!

**Comment by lunaryorn** [comment]
You're welcome.  Thank you for this contribution :)

**Pull request merged**